internal/provider/vm/vm_affinity_rule_schema.go

```diff
@@ -30,7 +30,7 @@ type vmAffinityRuleResourceModel struct {
 	Polarity types.String `tfsdk:"polarity"`
 	Required types.Bool   `tfsdk:"required"`
 	Enabled  types.Bool   `tfsdk:"enabled"`
-	VMIDs    types.List   `tfsdk:"vm_ids"`
+	VMIDs    types.Set    `tfsdk:"vm_ids"`
```

Comment:
ok, but why set type instead of list ?

Reply:
ok API return us a non order list

Reply:
Because the vCD API does not return the VMIDs in the given order